src/theme/default_theme.rs

```diff
@@ -126,7 +126,7 @@ impl UiStyles {
 
 impl Size {
     pub fn colourful(scale: ColorScaleOptions) -> Self {
-        if scale.size && scale.mode == ColorScaleMode::Fixed {
+        if scale.mode == ColorScaleMode::Fixed {
```

Comment:
here I really don't understand why we were checking if size scale was enabled (and if it was normal, why `scale.age` wasn't checked as well 🤔)

Reply:
Well iirc it was causing another bug but idrc